helm/auto-deployment/configuration.md

```diff
@@ -0,0 +1,444 @@
+       The <i>File share name</i> of your File share instance.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.azureStorageAccountName
+      </td>
+      <td>
+       The <i>base64 encoded storage account name</i> of the File share instance in your storage account.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.azureStorageAccountKey
+      </td>
+      <td>
+       The <i>base64 encoded storage account key</i> of the File share instance in your storage account.
+      </td>
+    </tr>
+</table>
+<br/>
+
+> **NOTE:** The Azure storage account credentials will be maintained in a secret named `bold-azure-secret`
+
+4. On-Premise
+  
+```console
+clusterProvider: onpremise
+    
+persistentVolume:
+  onpremise:
+    hostPath: /run/desktop/mnt/host/<local_directory>
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using. In this case the clusterProvider value is <i>onpremise</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.onpremise.hostPath
+      </td>
+      <td>
+       The shared folder path in your host machine.<br/><br/>
+       Ex: Create a folder in your machine to store the shared folders for application usage.
+       <i>D://app/shared</i><br/><br/>
+       The corresponding value for the above shared folder will be,
+       <i>/run/desktop/mnt/host/d/app/shared</i>
+      </td>
+    </tr>
+</table>
+<br/>
+
+## Load Balancing
+
+### Ingress-Nginx
+
+By default Nginx is used as reverse proxy for Bold BI. To configure load balancing in Bold BI take a look at the below snipet. 
+
+```console
+loadBalancer:
+  type: nginx
+```
+
+The default `ingressClassName` is `nginx`. Please refer [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation) for more details.
+
+```console
+loadBalancer:
+  # For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
+  # See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
+  # ingressClassName: nginx
+```
+
+### Istio Ingress Gateway
+
+If you need to configure Bold BI with Istio then you can change the value as `istio` in your configuration like below.
+
+```console
+loadBalancer:
+  type: istio
+```
+
+### sticky session
+
+You can change the affinityCookieExpiration time. The default value is 600s.
+
+```console
+loadBalancer:
+  affinityCookie:
+    enable: true
+    affinityCookieExpiration: 600
+```
+
+### SSL Configuration
+
+If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, just pass your DNS with `https` protocol to `appBaseUrl`, by doing this it will automatically enable SSL in both Ingress and Istio.

```

Comment:
If you have an SSL certificate for your DNS and need to configure the site with it, simply pass your DNS with the 'https' protocol to 'appBaseUrl'; this will enable SSL in both Ingress and Istio.

Reply:
concern addressed